icons/pronounsdotpage.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="24cm" height="24cm" viewBox="0 0 240 240"><path d="M149.873 51.535c.574-1.02.089-1.849-1.08-1.849l-20.489.013c-1.169 0-2.614.824-3.205 1.831l-50.203 84.874c-7.005 11.535-16.782 14.084-26.176 12.945-16.128-1.954-24.8-13.786-24.8-29.544 0-15.826 12.885-28.704 28.719-28.704H84.52c1.17 0 2.602-.828 3.18-1.843l11.49-20.054c.582-1.015.098-1.843-1.071-1.843H52.635C23.704 67.36.162 90.885.162 119.805c0 13.973 5.463 29.043 15.397 38.89 9.79 9.705 22.87 13.26 36.83 13.55 7.88.089 29.526-.09 39.672-18.133 11.363-20.21 57.812-102.577 57.812-102.577Zm74.568 29.778c-9.79-9.704-22.87-13.26-36.83-13.549-7.88-.09-29.526.09-39.672 18.133-11.359 20.202-57.812 102.568-57.812 102.568-.574 1.02-.09 1.849 1.08 1.849l20.489-.013c1.169 0 2.614-.824 3.205-1.831l50.203-84.874c7.005-11.535 16.782-14.084 26.176-12.945 16.128 1.954 24.8 13.787 24.8 29.544 0 15.826-12.885 28.704-28.719 28.704H155.48c-1.17 0-2.602.828-3.18 1.843l-11.49 20.054c-.582 1.015-.098 1.844 1.071 1.844h45.484c28.931 0 52.473-23.525 52.473-52.445 0-13.965-5.463-29.034-15.397-38.882z" style="stroke-width:.424969"/></svg>
```

Comment:
This file is massive - currently displying at 24x24cm, and 240x240px. Our guidelines stipulate that icons have to have no width or height attribute, and the viewBox should be 24x24px. Could you have another look at this? If it's playing up, usually has something to do with the DPI of your document and export being different!

Reply:
Hi sorry about that, think I uploaded the wrong one - will fix that now! Please ignore the last push :)